src/pages/careers/_employee-testimonial-carousel.js

```diff
@@ -74,20 +74,14 @@ const QuoteWrapper = styled(Flex)`
         top: 16px;
     }
 `
-const EmployeeSlide = ({ quote, img_path, img_alt, name }) => {
+const EmployeeSlide = ({ quote, img_path, name }) => {
     return (
         <Flex ai="center" height="unset">
             <EmployeeCard>
                 <Flex tablet_direction="column">
                     <Flex>
                         <ImageWrapper>
-                            <img
-                                loading="lazy"
-                                width="100%"
-                                height="100%"
-                                src={img_path}
-                                alt={img_alt}
-                            />
+                            <img loading="lazy" width="100%" height="100%" src={img_path} alt="" />
                         </ImageWrapper>
                     </Flex>
                     <QuoteWrapper direction="column">
```

Comment:
there's one more here 
'<img src={QuoteMark} alt="quotemark" />' line 88

Reply:
I didn't delete it because this image doesn't have any description

Reply:
Noted on this..